toxics/limit_data.go

```diff
@@ -0,0 +1,47 @@
+package toxics
+
+import "github.com/Shopify/toxiproxy/stream"
+
+// LimitDataToxic has limit in bytes
+type LimitDataToxic struct {
+	Bytes int64 `json:"bytes"`
+}
+
+func (t *LimitDataToxic) Pipe(stub *ToxicStub) {
+	var bytesRemaining = t.Bytes
```

Comment:
Alright, I confirmed this, and any fields in the toxic struct will be persisted until the toxic is removed.